specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1997,7 +1997,7 @@
           "type": "object",
           "properties": {
             "balance-similar-node-groups": {
-              "type": "boolean"
```

Comment:
please confirm this (whether it a issue which will be fixed with rebasing) else this change is a breaking change and would have to go in a  new api-version

Reply:
yup, that's a rebase issue:

The master branch is here: 

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json#L2000